### ISSUE TITLE
A bunch of favicons

### ISSUE THREAD

**Issue by var-bin** [var-bin/var-bin.com#110]
Added a bunch of favicons for all browsers and all platforms. Need to use the preparade list of favicons ([favicon_package_v0.16.zip](https://github.com/var-bin/var-bin.com/files/1860633/favicon_package_v0.16.zip)). Last but not least, [check it after adding](https://realfavicongenerator.net/favicon_checker)

---
1. Extract this package in the root of your website. If your site is `http://www.example.com`, you should be able to access a file named `http://www.example.com/favicon.ico`.
2. Insert the following code in the `<head>` section of your pages:
```html
<link rel="apple-touch-icon" sizes="76x76" href="/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
<link rel="manifest" href="/site.webmanifest">
<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#ffffff">
<meta name="theme-color" content="#ffffff">
```

**Comment by var-bin** [comment]
Good [service](https://favicon.io/favicon-generator) for generating text-based favicon

**Comment by var-bin** [comment]
Implemented in #152
